File: app/src/main/java/com/fast_prog/dynate/models/RegisterUser.java
package com.fast_prog.dynate.models;

import java.io.Serializable;

/**
 * Created by sarathk on 11/9/16.
 */

public class RegisterUser implements Serializable {

    //public String vTypeId;
    //public String vTypeName;

    public String name;
    public String nameArabic;
    public String mobile;
    public String mail;
    public String address;
    public String username;
    public String password;
    public String latitude;
    public String longitude;
    public String loginMethod;
    public String vModelId;
    public String vModelName;
    public String vCompId;
    public String vCompName;
    public String licenseNo;
    public String licenseNoArabic;

    public boolean withGlass;
}

File: app/src/main/java/com/fast_prog/dynate/models/RegisterUser.kt
package com.fast_prog.dynate.models

import java.io.Serializable

/**
 * Created by sarathk on 11/9/16.
 */

class RegisterUser : Serializable {

    var name: String? = null
    var nameArabic: String? = null
    var mobile: String? = null
    var mail: String? = null
    var address: String? = null
    //var username: String? = null
    //var password: String? = null
    var latitude: String? = null
    var longitude: String? = null
    var loginMethod: String? = null
    var vModelId: String? = null
    var vModelName: String? = null
    var vCompId: String? = null
    var vCompName: String? = null
    var licenseNo: String? = null
    var licenseNoArabic: String? = null
    var withGlass: Boolean = false
}
